docs/manual/guides/local-installation/devilbox.de.md

```diff
@@ -11,14 +11,10 @@ tags:
 ---
 
 Das [Devilbox Project](http://devilbox.org/) ist ein fertiges LAMP Stack für [Docker](https://www.docker.com/). 
-Wenn du die Docker-Toolbox einsetzt, sind [diese Angaben](https://devilbox.readthedocs.io/en/latest/howto/docker-toolbox/docker-toolbox-and-the-devilbox.html#howto-docker-toolbox-and-the-devilbox "Docker Toolbox and the Devilbox") der Dokumentation lesenswert.
-
-{{% notice note %}}
-Um die Devilbox nutzen zu können muss _Docker_ und _Docker Compose_ auf deinem System installiert sein. Falls das noch
-nicht der Fall ist, kannst du dir die 
-[Devilbox Prerequisites Dokumentation](https://devilbox.readthedocs.io/en/latest/getting-started/prerequisites.html) für 
-mehr Informationen zur Installation dieser Programme durchlesen.
-{{% /notice %}}
+Installationen der aktuellen »[Docker Desktop](https://www.docker.com/products/docker-desktop)« Version stehen für 
+verschiedene Betriebssysteme, u. a. auch für z. B. 
+»[Windows 10 Home](https://docs.docker.com/docker-for-windows/install-windows-home/)« oder 
+»[Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/)«, bereit. 
```

Comment:
This removes the notice about having to install Docker and Docker Compose and only mentions Windows.